coordinator/persistence-dse-next/pom.xml

```diff
@@ -18,7 +18,7 @@
     <!-- Latest as of 2023-12-05: 4.0.11-b259738f492f -->
     <!-- Latest as of 2024-01-24: 4.0.11-0248d170a615 -->
     <!-- Latest as of 2024-04-29: 4.0.11-591d171ac9c9 -->
-    <cassandra.version>4.0.11-591d171ac9c9</cassandra.version>
+    <cassandra.version>4.0.11-5696964ff12e</cassandra.version>
```

Comment:
When changing this version, please add a comment on date when update was made (to give an idea of how old DSE version is)

Reply:
do we need to update Cassandra ? IIUC we are only updating the dependencies in order to make it work on modern JDKs, the current version of Cassandra work decently on JDK22